hooks/frame_operations_tk-houdini.py

```diff
@@ -26,9 +26,12 @@ def get_frame_range(self, **kwargs):
         get_frame_range will return a tuple of (in_frame, out_frame)
 
         :returns: Returns the frame range in the form (in_frame, out_frame)
-        :rtype: tuple[int, int]
+        :rtype: tuple[float, float]
         """
-        current_in, current_out = hou.playbar.playbackRange()
+        # Get the global start and end frame
+        # Note, when using hou.playbar.playbackRange(), unpacking the start frame returns `1001.0000000000001`,
+        # thus will incorrectly flag the scene sync summary pop up to show, if the PTR start frame is 1001.0
+        current_in, current_out = hou.playbar.frameRange()
```

Comment:
Will this work instead?

```suggestion
        current_in, current_out = hou.playbar.frameRange()
        current_in = int(current_in)
        current_out = int(current_out)
```

I'm just trying to understand why we need to use floats for frames.